plugin/main/src/kotlinx/benchmark/gradle/BenchmarkConfiguration.kt

```diff
@@ -9,6 +9,7 @@ open class BenchmarkConfiguration(val extension: BenchmarksExtension, val name:
     var iterationTime: Long? = null
     var iterationTimeUnit: String? = null
     var mode: String? = null
+    var nativeGCCollectMode: String? = null
```

Comment:
I would rather make `nativeGCCollectMode` be declared via `advanced(...)` as well, similar to `nativeIterationMode`.